testsuite/gnat2goto/lib/python/test_support.py

```diff
@@ -91,16 +98,19 @@ def filter_cbmc_output(cbmc_output):
     lines = cbmc_output.split("\n")
     return "\n".join(filter(cbmc_match, lines))
 
-def prove(cbmcargs="", debug=False):
+def prove(cbmcargs="", main="", debug=False):
     """Call gnat2goto (and cbmc) on all *.adb files from the current directory
 
     PARAMETERS
       none: yet
```

Comment:
This docstring is lying about the parameters of this function.

Reply:
Added.